.github/workflows/build.yml

```diff
@@ -1,13 +1,44 @@
 name: GitHub CI
 on: push
 
+env:
+  dotnet_version: '3.0.100'
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
+
 jobs:
-  build-windows:
-    name: Build (Windows)
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, windows-latest ]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup .NET Core SDK
```

Comment:
.NET Core is already installed so this isn't needed

Reply:
Sure thing